src/main/java/com/google/cloud/tools/app/api/deploy/DeployConfiguration.java

```diff
@@ -28,17 +28,17 @@
 
   String getDockerBuild();
 
-  boolean isForce();
```

Comment:
I am also unsure about this comment. I guess it depends on the philosophy of how we map to gcloud.